### PR TITLE
Update System.Reflection.Metadata to 1.1.0-alpha-00008

### PR DESCRIPTION
**Pull request by tmat**

### CODE REVIEW

**Comment by tmat** [comment]
@jasonmalinowski @KevinH-MS @Pilchie @jaredpar

**Comment by KevinH-MS** [comment]
:+1:

File: src/VisualStudio/Core/Def/Implementation/EditAndContinue/VsENCRebuildableProjectImpl.cs [hunk 1]
```diff
@@ -30,6 +30,7 @@
 using ShellInterop = Microsoft.VisualStudio.Shell.Interop;
 using VsTextSpan = Microsoft.VisualStudio.TextManager.Interop.TextSpan;
 using VsThreading = Microsoft.VisualStudio.Threading;
+using Document = Microsoft.CodeAnalysis.Document;
```
Comment (jasonmalinowski) [inline src/VisualStudio/Core/Def/Implementation/EditAndContinue/VsENCRebuildableProjectImpl.cs]: Unrelated changes?

Comment (tmat) [inline src/VisualStudio/Core/Def/Implementation/EditAndContinue/VsENCRebuildableProjectImpl.cs]: No, these are needed to avoid ambiguity. There is now Document type in System.Reflection.Metadata.
